File: 백준/Bronze/2920. 음계/음계.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        int c = scanner.nextInt();
        int d = scanner.nextInt();
        int e = scanner.nextInt();
        int f = scanner.nextInt();
        int g = scanner.nextInt();
        int h = scanner.nextInt();

        if (a + h == 9 && b + g == 9 && c + f == 9 && d + e == 9) {
            if (a == 1 && b == 2 && c == 3 && d == 4) {
                System.out.println("ascending");
            } else if (a == 8 && b == 7 && c == 6 && d == 5) {
                System.out.println("descending");
            } else {
                System.out.println("mixed");
            }
        } else {
            System.out.println("mixed");
        }
    }
}

File: 백준/Bronze/2920. 음계/음계.py
a, b, c, d, e, f, g, h = map(int, input().split())

if a + h == 9 and b + g == 9 and c + f == 9 and d + e == 9:
    if a == 1 and b == 2 and c == 3 and d == 4:
        print("ascending")
    elif a == 8 and b == 7 and c == 6 and d == 5:
        print("descending")
    else:
        print("mixed")
else:
    print("mixed")
